onnxruntime/test/providers/cpu/math/element_wise_ops_test.cc

```diff
@@ -283,7 +283,7 @@ TEST(MathOpTest, Abs_int8) {
   std::vector<int64_t> dims{4};
   test.AddInput<int8_t>("X", dims, {1, 2, -1, -5});
   test.AddOutput<int8_t>("Y", dims, {1, 2, 1, 5});
-  test.Run();
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});
```

Comment:
this doesn't pass? I thought tensorrt can do int8.

Reply:
It's not because int8 is not supported in the parser. It's because parameter check failed in TRT, "Parameter check failed at: ../builder/Network.cpp::addInput::407, condition: type != DataType::kINT8". Need to ask Nvidia dev to look at the issue.